<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>schedule::TaskEditDialog</class>
 <widget class="QDialog" name="schedule::TaskEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task Editing</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="5">
    <widget class="QPlainTextEdit" name="textEdit"/>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>54</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QTimeEdit" name="timeEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="adv_settings_btn">
     <property name="toolTip">
      <string>Advanced task settings</string>
     </property>
     <property name="text">
      <string notr="true">...</string>
     </property>
     <property name="icon">
      <iconset resource="../schedule.qrc">
       <normaloff>:/schedule/notification.svg.p</normaloff>:/schedule/notification.svg.p</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../schedule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>schedule::TaskEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>schedule::TaskEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
